Azure/azure-cli
# [AppService] Support list, delete, approve and reject private endpoint connection for webapp in CLI

**Description**  

Closes #13072 

Add support for Webapp in the network private-endpoint-connection & private-link-resource commands

**Testing Guide**  

The existing commands:
`az network private-link-resource list`
`az network private-endpoint-connection (approve, delete, list, reject, show)`
Should now work with `--type Microsoft.Web/sites`

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
